scripts.js

```diff
@@ -44,6 +43,7 @@ function getRandomIndex(array) {
 }
 
 function createHexCode() {
+  var hexCharacters = ["A","B","C","D","E","F","0","1","2","3","4","5","6","7","8","9"];
```

Comment:
good catch, that makes sense to put the variable within the function instead of having it as a global variable.